README.md

```diff
@@ -275,3 +260,192 @@ datetime
 2018-06-04 00:00:00+00:00  0.07947329054545459  602.5116327272724
 
 ```
+
+### Exchange funds flow
+
+Fetch the difference between the tokens that were put in minus the tokens that were taken out from an exchange for a given slug in the selected time period.
+
+```python
+
+prices = san.get(
+    "exchange_funds_flow/santiment",
+    from_date="2018-04-16T10:02:19Z",
```

Comment:
I think we can make the examples only with dates: `from_date="2018-04-16"` - it will look easier to use ?

Reply:
Is it really that big of a difference if it's `2018-04-16T00:00:00Z` or `2018-04-16T10:02:19Z` and yes I'd like to leave it with time for the moment. This way we have places where we show only date and datetime.